pkg/resources/apiserver/deployment.go

```diff
@@ -410,6 +404,30 @@ func getApiserverFlags(data *resources.TemplateData, etcdEndpoints []string, ena
 	return flags, nil
 }
 
+// getApiserverOverrideFlags creates all settings that may be overridden by cluster specific componentsOverrideSettings
+// otherwise global overrides or defaults will be set
+func getApiserverOverrideFlags(data *resources.TemplateData) (settings kubermaticv1.APIServerSettings) {
+	// nodePortRange section
+	settings.NodePortRange = data.NodePortRange()
+	overrideNodePortRange := data.Cluster().Spec.ComponentsOverride.Apiserver.NodePortRange
+	if overrideNodePortRange != "" {
+		if _, err := net.ParsePortRange(overrideNodePortRange); err == nil {
```

Comment:
Why ignoring the error completely? If the default NodePortRange will be the first fallback, I guess it makes sense to at least log the error.

Reply:
Ah good catch.
Thinking about it, it is probably better to just throw an error here to make it a bit clearer.
And I can't think of a reason why someone would like to have the default on if the custom port range is invalid 🤔

Reply:
>Thinking about it, it is probably better to just throw an error here to make it a bit clearer.

Than please remove the naked return along the way ;)

Reply:
@phiphi282  You are returning the error when err == nil :-)